### PR TITLE
feat question pages

### PR DESCRIPTION
**Pull request by AmishMishra11**
https://user-images.githubusercontent.com/76205249/154233723-531b65a5-5e37-482b-9a17-3c85f82c6a14.mp4

### CODE REVIEW

File: stylesheet/default.css [hunk 1]
```diff
@@ -32,6 +34,9 @@ body {
 
   /* border radius */
   --br-S: 5px;
+
+  --br-M: 10px;
+
```
Comment (Prabhav-Sharma) [inline stylesheet/default.css]: Remove empty lines of code

**Review by Prabhav-Sharma — APPROVED** [review]

**Review by SukanyaSen530 — COMMENTED** [review]
Suggestion : Generally red is used to show error or wrong answer and green for correct, Better if you change the color of next and previous buttons.

File: stylesheet/default.css (hunk 1 above)
Comment (AmishMishra11) [reply stylesheet/default.css]: ok

**Comment by AmishMishra11** [comment]
> Suggestion : Generally red is used to show error or wrong answer and green for correct, Better if you change the color of next and previous buttons.

Thanks for the suggestion, I will look into it after I've completed the app.

**Pull request merged**